pom.xml

```diff
@@ -0,0 +1,384 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>se.lfv.reqstool</groupId>
+    <artifactId>reqstool-java-annotations</artifactId>
+    <version>0.0.6</version>
+    <packaging>jar</packaging>
+
+    <name>${project.artifactId}</name>
+    <description>Requirements Tools - Java Annotations</description>
+    <url>https://github.com/Luftfartsverket/reqstool-java-annotations</url>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://www.opensource.org/licenses/mit-license.php</url>
```

Comment:
If you click on the link you'll redirect (without php) - use that instead.

Reply:
yep!